relayer/provider/cosmos/query.go

```diff
@@ -77,26 +77,28 @@ func (cc *CosmosProvider) QueryTxs(ctx context.Context, page, limit int, events
 	// at most, one tx in the response. Because of this we don't want to initialize the slice with an initial size.
 	var txResps []*provider.RelayerTxResponse
 	for _, tx := range res.Txs {
-		events := parseEventsFromResponseDeliverTx(tx.TxResult)
+		relayerEvents := parseEventsFromResponseDeliverTx(tx.TxResult)
 		txResps = append(txResps, &provider.RelayerTxResponse{
 			Height: tx.Height,
 			TxHash: string(tx.Hash),
 			Code:   tx.TxResult.Code,
 			Data:   string(tx.TxResult.Data),
-			Events: events,
+			Events: relayerEvents,
 		})
 	}
 	return txResps, nil
 }
 
-// parseEventsFromResponseDeliverTx parses the events from a ResponseDeliverTx and builds a map
-// where the keys are equal to event.Type+"."+attribute.Key and the values are the attribute.Value.
-func parseEventsFromResponseDeliverTx(resp abci.ResponseDeliverTx) (events map[string]string) {
-	events = make(map[string]string, len(resp.Events))
+// parseEventsFromResponseDeliverTx parses the events from a ResponseDeliverTx and builds a slice
+// of provider.RelayerEvent's.
+func parseEventsFromResponseDeliverTx(resp abci.ResponseDeliverTx) (events []*provider.RelayerEvent) {
```

Comment:
Personally, I find the bare return on L104 a bit less readable as opposed to omitting the name of the return value and changing to an explicit `return events` instead. But it is correct either way.